presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java

```diff
@@ -153,34 +151,34 @@ public PipelineStats(
         checkArgument(completedDrivers >= 0, "completedDrivers is negative");
         this.completedDrivers = completedDrivers;
 
-        this.userMemoryReservation = requireNonNull(userMemoryReservation, "userMemoryReservation is null");
-        this.revocableMemoryReservation = requireNonNull(revocableMemoryReservation, "revocableMemoryReservation is null");
-        this.systemMemoryReservation = requireNonNull(systemMemoryReservation, "systemMemoryReservation is null");
+        this.userMemoryReservationInBytes = requireNonNull(userMemoryReservationInBytes, "userMemoryReservationInBytes is null");
```

Comment:
primitives can not be null. no need to check.

Reply:
done